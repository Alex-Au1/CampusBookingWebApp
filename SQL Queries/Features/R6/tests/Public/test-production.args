{
    "args": [],
    "kwargs": {
        "roomName": "CAPB3 1",
        "minCapacity": "100",
        "maxCapacity": "200",
        "startTimeStr": "2025-06-23 17:46:05",
        "endTimeStr": "2025-06-23 20:07:05"
    }
}   