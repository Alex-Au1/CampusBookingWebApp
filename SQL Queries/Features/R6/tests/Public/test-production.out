{'roomID': UUID('00000000-0000-0000-0000-000000000000'), 'buildingName': 'Carl A. Pollock Building 3', 'roomName': 'CAPB3 1', 'overlappingBookings': 0, 'capacity': 175, 'addressLine1': '966 Philips St E', 'addressLine2': '', 'city': 'Waterloo', 'province': 'Ontario', 'country': 'Canada', 'postalCode': 'L4H 2X4'}
{'roomID': UUID('00000000-0000-0000-0000-000000001353'), 'buildingName': 'Carl A. Pollock Building 3', 'roomName': 'CAPB3 10', 'overlappingBookings': 0, 'capacity': 175, 'addressLine1': '966 Philips St E', 'addressLine2': '', 'city': 'Waterloo', 'province': 'Ontario', 'country': 'Canada', 'postalCode': 'L4H 2X4'}